keda/templates/22-metrics-deployment.yaml

```diff
@@ -18,6 +20,8 @@ spec:
       app: {{ .Values.operator.name }}-metrics-apiserver
   template:
     metadata:
+      annotations:
+        {{- toYaml .Values.additionalAnnotations | nindent 8 }}
```

Comment:
move this to the already existing `annotations` block

Reply:
Done